apps/tangent-electron/src/app/model/directoryView.ts

```diff
@@ -35,13 +34,11 @@ function encodeSortMode({ key, order }: SortMode): string {
 }
 
 export function decodeSortMode(value: string): SortMode {
-	if (typeof value === 'string') {
```

Comment:
There was definitely a reason I did this. I wonder if it was some legacy information. In any case, it was 4 years ago. This change is fine now.